pkg/kubectl/cmd/proxy.go

```diff
@@ -35,21 +35,10 @@ import (
 var (
 	defaultPort = 8001
 	proxyLong   = templates.LongDesc(i18n.T(`
-		To proxy all of the kubernetes api and nothing else, use:
-
-		    $ kubectl proxy --api-prefix=/
-
-		To proxy only part of the kubernetes api and also some static files:
-
-		    $ kubectl proxy --www=/my/files --www-prefix=/static/ --api-prefix=/api/
-
-		The above lets you 'curl localhost:8001/api/v1/pods'.
-
-		To proxy the entire kubernetes api at a different root, use:
-
-		    $ kubectl proxy --api-prefix=/custom/
-
-		The above lets you 'curl localhost:8001/custom/api/v1/pods'`))
+		Creates a proxy server or application-level gateway between local host and 
+		kubernetes apiserver; It also allows serving static content over specified 
```

Comment:
`apiserver`, when used in text, should be `API Server`:

    kubectl --help | grep proxy